File: 프로그래머스/lv0/120888. 중복된 문자 제거/중복된 문자 제거.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

string solution(string my_string)
{
    string a = my_string;

    for (int i = 0; i < a.length(); i++)
        a.erase(remove(a.begin() + i + 1, a.end(), a[i]), a.end());

    return a;
}

File: 프로그래머스/0/120888. 중복된 문자 제거/중복된 문자 제거.java
import java.util.Arrays;
import java.util.LinkedHashSet;
import java.util.Set;
import java.util.stream.Collectors;

class Solution {
    public String solution(String my_string) {
        Set<String> set = new LinkedHashSet<>(Arrays.stream(my_string.split("")).collect(Collectors.toList()));
        return String.join("", set);
    }
}